content/blog/2018-02-10-drake.md

```diff
@@ -0,0 +1,117 @@
+---
+slug: "drake"
+title: "The prequel to the drake R package"
+package_version: 5.0.0
+authors:
+  - name: Will Landau
+    url: https://github.com/wlandau
+date: 2018-02-10
+categories: blog
+topicid: 1033
+tags:
+- r
+- community
+- software
+- review
+- onboarding
+- package
+- drake
+- reproducibility
+- high-performance-computing
+- pipeline
+- workflow
+output:
+  html_document:
+    keep_md: yes
+---
+
+The [drake](https://github.com/ropensci/drake) package is a [pipeline](https://github.com/pditommaso/awesome-pipeline) toolkit. It manages data science workflows, [saves time](https://ropensci.github.io/drake/#what-gets-done-stays-done-), and [ensures a kind of reproducibility](https://ropensci.github.io/drake/#reproducibility-with-confidence) that the R community seldom thinks about. I am trying to change the conversation around reproducible research, and I am using [drake](https://github.com/ropensci/drake) as a flagship. However, I originally created [drake](https://github.com/ropensci/drake) for totally different reasons. This post is the prequel to [drake](https://github.com/ropensci/drake)'s inception. There was struggle, and [drake](https://github.com/ropensci/drake) was the answer.
```

Comment:
I'm concerned about how this part sounds: "...ensures a kind of reproducibility that the R community seldom thinks about. I am trying to change the conversation around reproducible research ..."

The first part isn't necessarily an accurate reflection on the R community as it is today.  This may be your personal experience and that's always a useful viewpoint, however, many people are actively doing and thinking about that and many have been trying to change the conversation for a while. I would consider how to rephrase to make that more clear, taking care not to offend potential users or fans of `drake` before they try it out. Consider it a way to engage with precisely the kinds of people you want to work with and among in order to make the change.